.github/workflows/ci.yml

```diff
@@ -63,6 +63,34 @@ jobs:
           name: ${{ matrix.python-version }}
           fail_ci_if_error: false
 
+  benchmark:
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.LATEST_PY_VERSION }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .["all,test"]
+
+      - name: Run Benchmark
+        run: python -m pytest --benchmark-only --benchmark-autosave --benchmark-columns 'min, max, mean, median' --benchmark-json output.json
+
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
```

Comment:
never used this, let see how it goes!

Reply:
`github-action-benchmark` is a nice gh action (https://github.com/benchmark-action/github-action-benchmark) which can create HTML page to follow the evolution of the performance see : https://vincentsarago.github.io/lebanc/benchmarks/